scripts/privacyidea

```diff
@@ -765,7 +769,7 @@ def machine_attachtoken(args, client):
         options = options_to_dict(args.option)
         ret = client.connect("/machine/addoption",
                              {},
-                             dict(param.items() + options.items()))
+                             param.update(options))
```

Comment:
Ah, I think this doesn't to the right thing because ``.update`` modifies the dictionary in-place and returns ``None``. So this will pass ``None`` to ``client.connect``

Reply:
Yes, You are right. We need tests :-)

Reply:
fixed in 13670ac